### PR TITLE
Add a github action to run cargo-flaky 1000 times

### PR DESCRIPTION
**Pull request by irevoire**
I don’t know how to ensure the CI works so it’s just a first version, do not hesitate to update the code

### CODE REVIEW

File: .github/workflows/flaky.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  schedule:
+    - cron: "* */12 * * */5" # every friday at 12PM
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: "install cargo-flaky"
```
Comment (curquiza) [inline .github/workflows/flaky.yml:12]:
```suggestion
    - name: Install cargo-flaky
```

File: .github/workflows/flaky.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: "Look for flaky tests"
+on:
+  schedule:
+    - cron: "* */12 * * */5" # every friday at 12PM
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (curquiza) [inline .github/workflows/flaky.yml:8]:
```suggestion
    runs-on: ubuntu-18.04
```

Comment (curquiza) [reply .github/workflows/flaky.yml:8]: Same as transplant CI test

File: .github/workflows/flaky.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+    steps:
+    - uses: actions/checkout@v1
```
Comment (curquiza) [inline .github/workflows/flaky.yml:11]:
```suggestion
    - uses: actions/checkout@v2
```

File: .github/workflows/flaky.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    - name: "install cargo-flaky"
+      run: cargo install cargo-flaky
+    - name: "Run cargo flaky 1000 times"
```
Comment (curquiza) [inline .github/workflows/flaky.yml:14]:
```suggestion
    - name: Run cargo flaky 1000 times
```

File: .github/workflows/flaky.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+name: "Look for flaky tests"
```
Comment (curquiza) [inline .github/workflows/flaky.yml:1]:
```suggestion
name: Look for flaky tests
```

**Review by curquiza — CHANGES_REQUESTED** [review]

File: .github/workflows/flaky.yml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+name: Look for flaky tests
+on:
+  schedule:
+    - cron: "* */12 * * */5" # every friday at 12PM
+
+jobs:
+  build:
```
Comment (curquiza) [inline .github/workflows/flaky.yml:7]:
```suggestion
  flaky:
```

**Comment by irevoire** [comment]
@MarinPostma since you wrote the tool I added you as a reviewer so you can add some args to cargo-flaky.
Also we should implement this issue https://github.com/MarinPostma/cargo-flaky/issues/7

File: .github/workflows/flaky.yml [hunk 7]
```diff
@@ -11,4 +11,4 @@ jobs:
     - name: Install cargo-flaky
       run: cargo install cargo-flaky
     - name: Run cargo flaky 1000 times
-      run: cargo flaky -i 1 --release
+      run: cd meilisearch-http; cargo flaky -i 1 --release
```
Comment (curquiza) [inline .github/workflows/flaky.yml:14]:
```suggestion
      run: |
        cd meilisearch-http
        cargo flaky -i 1 --release
```

Comment (irevoire) [reply .github/workflows/flaky.yml]: cargo-flaky has been patched, we now don’t need to cd before running the command

**Comment by irevoire** [comment]
If you are ok with the PR we can merge 😄

**Review by MarinPostma — APPROVED** [review]
Bors merge

File: .github/workflows/flaky.yml (hunk 7 above)
Comment (MarinPostma) [reply .github/workflows/flaky.yml]: We need to create a release for that

**Review by curquiza — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Comment by Kerollmops** [comment]
Quick question: How would you be informed if the test fail?

**Comment by curquiza** [comment]
If I understand well how GH works, it depends on:
- if you watch the repo
- how you set your notification settings:
Capture d’écran 2021-06-14 à 09 39 15

In my case, I hope I will receive a GH notification :)

**Pull request merged**